subprojects/docs/src/docs/release/notes.md

```diff
@@ -54,6 +54,31 @@ The allowed labels are:
 
 More details can be found in the [Gradle Wrapper](userguide/gradle_wrapper.html#sec:adding_wrapper) section.
 
+### Kotlin DSL
+
+#### Experimental lazy property assignment for Kotlin scripts
+
+It is now possible to use the `=` operator to assign values to `Property` types in Kotlin scripts as an alternative to the `set()` method:
```

Comment:
💭 
Is there a benefit for one way or the other? Why would we want to change?

Reply:
This is detailed just below the example.